lib/ohai/dsl/plugin.rb

```diff
@@ -82,6 +83,10 @@ class Plugin
       include Ohai::Mixin::Command
       include Ohai::Mixin::SecondsToHuman
 
+      # Pending next major release of Ohai.
```

Comment:
I'd be fine if we kept this part of the Plugin base API. It's small and useful for writing plugins.